File: artifacts/lodepng/translate_without_break_down/src/lode_p_n_g_decoder_settings.rs
use crate::*;
use std::os::raw::c_uint;



pub struct LodePNGDecoderSettings {
    pub zlibsettings: LodePNGDecompressSettings,
    pub ignore_crc: c_uint,
    pub ignore_critical: c_uint,
    pub ignore_end: c_uint,
    pub color_convert: c_uint,
    pub read_text_chunks: c_uint,
    pub remember_unknown_chunks: c_uint,
    pub max_text_size: usize,
    pub max_icc_size: usize,
}

File: artifacts/lodepng/decompose/src/lode_p_n_g_decoder_settings.c
typedef struct LodePNGDecoderSettings
{
  LodePNGDecompressSettings zlibsettings;
  unsigned ignore_crc;
  unsigned ignore_critical;
  unsigned ignore_end;
  unsigned color_convert;
  unsigned read_text_chunks;
  unsigned remember_unknown_chunks;
  size_t max_text_size;
  size_t max_icc_size;
} LodePNGDecoderSettings
/*** DEPENDENCIES:
struct LodePNGDecompressSettings
{
  unsigned ignore_adler32;
  unsigned ignore_nlen;
  size_t max_output_size;
  unsigned (*custom_zlib)(unsigned char **, size_t *, const unsigned char *, size_t, const LodePNGDecompressSettings *);
  unsigned (*custom_inflate)(unsigned char **, size_t *, const unsigned char *, size_t, const LodePNGDecompressSettings *);
  const void *custom_context;
}
----------------------------
***/


